Cargo.toml

```diff
@@ -18,10 +18,10 @@ rust-version = "1.56.0"
 arc-swap = "~1"
 # It's OK to ask for std on log, because pyo3 needs it too.
 log = { version = "~0.4.4", default-features = false, features = ["std"] }
-pyo3 = { version = ">=0.21, <0.22", default-features = false }
+pyo3 = { version = ">=0.21, <=0.22", default-features = false }
```

Comment:
I think this'll disallow 0.22.1 (for example), while `<0.23` (following the previous style) would allow it. And I assume it is a good thing to allow the patch versions that aren't supposed to break things.

Reply:
Oops I hadn't thought about this disallowing 0.22.1